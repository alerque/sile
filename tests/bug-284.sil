\begin[papersize=a6]{document}
\nofolios
\set[parameter=document.lskip,value=15pt]
\set[parameter=document.rskip,value=10pt]
\font[family=Gentium Plus,size=11pt,language=tr]
\noindent
Çünkü Müjde’den utanmıyorum. Müjde iman eden herkesin –önce Yahudiler’in, sonra Yahudi olmayanların– kurtuluşu için Tanrı gücüdür.
\end{document}
