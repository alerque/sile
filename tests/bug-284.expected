Set paper size 	297.6377985	419.5275636
Begin page
Mx 	29.8819
My 	29.3016
Set font 	Gentium Plus;11;400;;normal;;LTR
T	100 129 81 78 129	(Çünkü)
Mx 	60.9691
T	48 129 77 71 72 183 71 72 81	(Müjde’den)
Mx 	110.8014
T	88 87 68 81 80 215 92 82 85 88 80	(utanmıyorum)
Mx 	173.3263
T	17	(.)
Mx 	177.9447
T	48 129 77 71 72	(Müjde)
Mx 	208.3873
T	76 80 68 81	(iman)
Mx 	233.4213
T	72 71 72 81	(eden)
Mx 	257.4723
T	75 72 85	(her)
Mx 	272.9787
T	16	(-)
Mx 	29.8819
My 	42.5016
T	78 72	(ke)
Mx 	40.3287
T	86 76 81	(sin)
Mx 	55.7896
T	178	(–)
Mx 	60.9512
T	124 81 70 72	(önce)
Mx 	84.6031
T	60 68 75 88 71 76 79 72 85 183 76 81	(Yahudiler’in)
Mx 	140.7740
T	15	(,)
Mx 	145.4551
T	86 82 81 85 68	(sonra)
Mx 	172.8721
T	60 68 75 88 71 76	(Yahudi)
Mx 	207.2178
T	82 79 80 68 92 68 81 79 68 85 215 81	(olmayanların)
Mx 	267.5943
T	178	(–)
Mx 	29.8819
My 	55.7016
T	78 88 85 87 88 79 88 1578 88	(kurtuluşu)
Mx 	76.8502
T	76 111 76 81	(için)
Mx 	96.6086
T	55 68 81 85 215	(Tanrı)
Mx 	124.0422
T	74 129 70 129 71 129 85	(gücüdür)
Mx 	161.7741
T	17	(.)
End page
Finish
